rest/utilities_testing.go

```diff
@@ -351,6 +351,29 @@ func (rt *RestTester) WaitForNViewResults(numResultsExpected int, viewUrlPath st
 
 }
 
+func (rt *RestTester) WaitForDBOnline() (err error) {
+
+	maxTries := 50
```

Comment:
I feel like 25 seconds (total) is a bit long of a wait - if the DB is actually taking that long to come online, even under -race, it feels like there might be a bug?

Reply:
Yeah, I err on the side of padding since I feel like these kinds of failures are usually just noise, but I'll reduce it to 10s since 25s is a bit excessive